example/Component.php

```diff
@@ -4,6 +4,8 @@
 
 namespace MyComponent;
 
+use Keboola\Component\Manifest\ManifestManager\Options\WriteTableManifestOptions;
```

Comment:
co takhle vyhodit namespace `options` ?

Reply:
a nemelo by teda existovat i WriteFileManifestOptions ? kdyz uz jsme u tech BC breaku

Reply:
jeste uvazuju jestli by se ta trida nemohla jmenovat `OutTableManifestOptions`

Reply:
options by mělo sdružovat ty optiony, ať jsou na jednom místě...

Reply:
jo, ja to prvni napsal jeste driv nez me napadlo, ze bude filemanifestoptions :)